covid-nlp/main.ipynb

```diff
@@ -0,0 +1,3181 @@
+{
```

Comment:
<p>what do you mean by excluding generic queries?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='94'/>

Reply:
<p>generic is meant only for generic ie functions (not related to queries), what we meant here is that we added all rgxlog code except quires since they don't modify the data or contribute to the code flow, and they are only for reading contents (we included only querying the final result which is stored in DataFrame). for example:</p><p>before:</p><pre class="ql-syntax" spellcheck="false">%%rgxlog
FilesContent(Path, Content) &lt;- FilesPaths(Path), read_from_file(Path) -&gt; (Content)
?FilesContent(Path, Content)
</pre><p>after:</p><pre class="ql-syntax" spellcheck="false">%%rgxlog
FilesContent(Path, Content) &lt;- FilesPaths(Path), read_from_file(Path) -&gt; (Content)
</pre><p> </p><p>Nevertheless, do you think it's advisable to include them as well?</p>

Reply:
<p>now i understand, do exclude generic ie function and change the wording of this cell to make it clear there is no such thing as generic query.</p>